Source/objects.cpp

```diff
@@ -5255,66 +5265,69 @@ int ItemMiscIdIdx(item_misc_id imiscid)
 
 void OperateObject(int pnum, int i, bool teleFlag)
 {
+	Player &player = Players[pnum];
 	bool sendmsg = pnum == MyPlayerId;
-	switch (Objects[i]._otype) {
+	Object &object = Objects[i];
+
+	switch (object._otype) {
 	case OBJ_L1LDOOR:
 	case OBJ_L1RDOOR:
 		if (teleFlag) {
-			if (Objects[i]._otype == OBJ_L1LDOOR)
-				OperateL1LDoor(pnum, i, true);
-			if (Objects[i]._otype == OBJ_L1RDOOR)
-				OperateL1RDoor(pnum, i, true);
+			if (object._otype == OBJ_L1LDOOR)
+				OperateL1LDoor(&player, i, true);
+			if (object._otype == OBJ_L1RDOOR)
+				OperateL1RDoor(&player, i, true);

```

Comment:
These calls could use `sendflag` as declared a few lines up instead of passing through the player reference.